File: 039/RecoverRotatedSortedArray.py
class Solution:
    """
    @param nums: The rotated sorted array
    @return: nothing
    """
    def recoverRotatedSortedArray(self, nums):
        # write your code here
        for i in range(len(nums)):
            for j in range(i+1, len(nums)):
                if nums[i] > nums[j]:
                    temp = nums[i]
                    nums[i] = nums[j]
                    nums[j] = temp

File: 039 恢复旋转排序数组/RecoverRotatedSortedArray.java
import java.lang.reflect.Array;
import java.util.ArrayList;
import java.util.Arrays;

/**
 * Created by Silocean on 2016-04-01.
 */
public class RecoverRotatedSortedArray {
    public static void main(String[] args) {
        ArrayList<Integer> nums = new ArrayList<>(Arrays.asList(7, 8, 9, 1, 2, 3, 4, 5, 6));
        recoverRotatedSortedArray(nums);
    }

    public static void recoverRotatedSortedArray(ArrayList<Integer> nums) {
        // write your code
        while (nums.get(0) >= nums.get(nums.size()-1)) {
            int temp = nums.remove(0);
            nums.add(temp);
        }
        /*for (int i = 0; i < nums.size(); i++) {
            System.out.println(nums.get(i));
        }*/
    }
}
